travis/build-wheels.sh

```diff
@@ -63,7 +57,7 @@ done
 # Fetch sources
 ############################
 # we use 2.7 for downloads
-export PIPBIN=/opt/python/cp27-cp27m/bin/pip
+export PIPBIN=/opt/python/cp37-cp37m/bin/pip
```

Comment:
Why use 37 for pip? I would rather stick to 36

Reply:
doing now